ApiDemos/java/app-v3/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,242 @@
+<?xml version="1.0" encoding="utf-8"?><!--
```

Comment:
It would be nice to share resources between the two sets of examples. Build flavors instead of a separate module is one way to do this, and if we're not publishing the artifacts we wouldn't have to wrestle with that configuration here.

Reply:
Agreed that product flavor would be better here. Will update.